internal/pkg/task/task.go

```diff
@@ -40,6 +40,13 @@ const (
 
 var (
 	errNoSubnetFound  = errors.New("no subnets found")
+
+	errVPCGetterNil = errors.New("vpc getter is not set")
+	errClusterGetterNil = errors.New("cluster getter is not set")
+	errStarterNil = errors.New("starter is not set")
+
+	fmtErrDefaultCluster = "get default cluster: %w"
```

Comment:
Maybe we can also have a struct instead of just the string constant:
```go
type errGetDefaultCluster struct {
   parentErr error
}
func (e *errGetDefaultCluster) Error() string { return fmt.Sprintf("get default cluster: %v, e.parentErr) }
```